src/_global.scss

```
@@ -50,6 +50,7 @@ h2 {
 
 a {
   @include tr(color 0.6s, background-size 0.6s);
+  overflow-wrap: anywhere;
```

```suggestion
  overflow-wrap: break-word;
```
Safari doesn't support the `anywhere` value.
It seems that `break-word` works everywhere, though.